performance/README.md

```diff
@@ -0,0 +1,26 @@
+# Performance Testing with Gatling
+
+* [Gatling Documentation](https://gatling.io/docs/current/)
+* [Gradle Plugin Documentation](https://github.com/lkishalmi/gradle-gatling-plugin)
+
+## Running Tests
+
+From the `performance` directory, run the gatling tests (optionally accepts `-Denv=<ENV>` 
+which defaults to the test environment):
```

Comment:
nit: could you give an example value here? e.g. "test" (or is the cloud project id?)

Reply:
Added some more info here, hopefully it is clearer.